_data/cryptocurrencies.yml

```
@@ -182,6 +182,14 @@ websites:
       software: Yes
       doc: https://support.circle.com/hc/en-us/articles/213560483--What-is-two-factor-authentication-2FA-
 
+    - name: COBINHOOD
+      url: https://cobinhood.com
+      img: cobinhood.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://cobinhood.zendesk.com/hc/en-us/articles/115003476871-Setting-up-two-factor-authentication-2FA-
```

When linking to Zendesk links, only the numbers of the article ID are needed.
```suggestion
      doc: https://cobinhood.zendesk.com/hc/en-us/articles/115003476871
```